index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="stylesheet" href="style.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="bar-nav">
+        <h1>My Logo</h1>
+        <img src="image/Union.png" alt="Menu icon">
+    </div>
+    <div class="container">
+      <section class="detail one">
+        I'm Mariane<br>Glad to see<br>you
+      </section>
+      <section class="detail two">
+        I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience. If you like what you see and have a project you need code, don't hesitate to contact me
+      </section>
+      <section class="detail link">
+        <a href="#">LET'S CONNECT</a>
+      </section>
```

Comment:
- [ ] Please try to use the `section` tag just to wrap a section not just a small part of the code, check this [link](https://www.w3schools.com/tags/tag_section.asp) for more information, Kindly remove the `section` tag from here

Reply:
I have done it